src/models/request.ts

```diff
@@ -15,6 +15,7 @@ export interface Request {
     restrictNavigation: RestrictNavigation;
     biasing: Biasing;
     matchStrategy: MatchStrategy;
+    ['pre-filter']?: string;
```

Comment:
The brackets are not needed since the value inside is a string:
```suggestion
    'pre-filter'?: string;
```